src/containers/index.js

```diff
@@ -1,13 +1,16 @@
-import React, { Component } from 'react';
-import { Provider } from 'react-redux';
-import store  from '../js/store/index';
+import React, { Component } from "react";
 import Routes from "../components/routes";
+import Navbar from "../components/navBar";
 
- class App extends Component{
-    render() {
-        return(
-                <Routes/>
-        );
-    }
+class App extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
@magicmarie why did you include all these divs, you should only render <Routes /> here